mappings/com/mojang/blaze3d/MemoryUtil.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_ilpgzdyq net/minecraft/client/util/GlAllocationUtils
+CLASS net/minecraft/unmapped/C_ilpgzdyq com/mojang/blaze3d/MemoryUtil
```

Comment:
MemoryUtil is used in OpenGL but I assume this is just a class mojang added that has the MemoryUtil methods from opengl in it

Reply:
`NativeByteBufferUtils`?